.github/workflows/camunda-load-test.yml

```diff
@@ -297,9 +297,10 @@ jobs:
           --reuse-values
           --render-subchart-notes
           --set global.image.tag=${{ needs.calculate-image-tag.outputs.image-tag }}
-          --set orchestration.image.registry=gcr.io
-          --set orchestration.image.repository=zeebe-io/zeebe
-          --set orchestration.image.tag=${{ needs.calculate-image-tag.outputs.image-tag }}
+          --set zeebe.image.repository=gcr.io/zeebe-io/zeebe
```

Comment:
💭 in the helm chart for the previous versions we need to set "old" `zeebe` keys as `orchestration` namespace is only for 8.8+